### ISSUE TITLE
InsertImage Bug

### ISSUE THREAD

**Issue by houzixiashanxiedaima** [zhuochun/md-writer#183]
can not insert image .
The error is `Uncaught TypeError: Cannot read property 'showOpenDialog' of undefined`
here is the Bug Report:
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.12.1
**Electron Version**: 1.3.6
**System**: Unknown Windows Version
**Thrown From**: [markdown-writer](https://github.com/zhuochun/md-writer) package, v2.6.1

### Stack Trace

Uncaught TypeError: Cannot read property 'showOpenDialog' of undefined

```
At /X:/Users/Stephen/.atom/packages/markdown-writer/lib/views/insert-image-view.coffee:241

TypeError: Cannot read property 'showOpenDialog' of undefined
    at InsertImageView.module.exports.InsertImageView.openImageDialog (file:///X:/Users/Stephen/.atom/packages/markdown-writer/lib/views/insert-image-view.coffee:116:19)
    at HTMLButtonElement.<anonymous> (file:///X:/Users/Stephen/.atom/packages/markdown-writer/lib/views/insert-image-view.coffee:54:38)
    at HTMLButtonElement.dispatch (X:\Users\Stephen\AppData\Local\atom\app-1.12.1\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
    at HTMLButtonElement.elemData.handle (X:\Users\Stephen\AppData\Local\atom\app-1.12.1\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)
```

### Commands

```
     -3:25.2.0 markdown-writer:insert-new-line (atom-text-editor.editor.is-focused)
     -3:24.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -3:12.5.0 markdown-writer:insert-image (atom-text-editor.editor.is-focused)
     -2:15.4.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
  2x -2:15.4.0 core:cancel (atom-text-editor.editor.is-focused)
     -1:31.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:06.0 markdown-writer:insert-image (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "closeEmptyWindows": false,
    "destroyEmptyPanes": false,
    "packagesWithKeymapsDisabled": [],
    "telemetryConsent": "limited",
    "themes": [
      "atom-material-ui",
      "one-dark-syntax"
    ]
  },
  "markdown-writer": {
    "renameImageOnCopy": true,
    "tableExtraPipes": true
  }
}
```

### Installed Packages

```coffee
# User
atom-material-ui, v1.3.6 (active)
file-icons, v1.7.24 (active)
markdown-scroll-sync, v2.1.2 (active)
markdown-writer, v2.6.1 (active)
monokai-flat, v1.0.1 (inactive)
seti-ui, v1.3.2 (inactive)
simplified-chinese-menu, v3.5.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (inactive)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (active)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.202.2 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.1.1 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.4 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.0 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.12.1 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.46.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by zhuochun** [comment]
Thanks, looks like compatibility issue with new Atom v1.12.1. Will investigate

**Comment by zhuochun** [comment]
Fixed in https://github.com/zhuochun/md-writer/commit/e5e3c78307af061f508412c15dd6bc9398477f6a

**Comment by zhuochun** [comment]
Released in v2.6.2
